src/lib/components/collections/req-res-section/sub-components/close-popup/ClosePopup.svelte

```diff
@@ -139,20 +139,21 @@
       </div>
       <div class="d-flex">
         <span style="margin-right: 15px;">
-          <CustomButton
-            text={"Discard Changes"}
-            fontSize={16}
+          <Button
+            title={"Discard Changes"}
+            textStyleProp={"font-size: 16px"}
             type={"dark"}
             onClick={() => {
               collectionsMethods.handleRemoveTab(componentData.id);
               closeCallback(false);
             }}
           />
         </span>
-        <CustomButton
-          text={"Save Changes"}
-          fontSize={16}
+        <Button
+          title={"Save Changes"}
+          textStyleProp={"font-size: 16px"}
```

Comment:
do it wherever required for font size

Reply:
Same, using rem from global css for 12 and 14 pixel and fs-6 for 16px.